pulp_rpm/app/viewsets.py

```diff
@@ -316,6 +318,44 @@ class DistributionTreeViewSet(ReadOnlyContentViewSet,
     serializer_class = DistributionTreeSerializer
 
 
+class DistributionTreeRepositoryViewSet(RepositoryViewSet):
```

Comment:
I believe this will expose it to the user, and this is what we tried to avoid, since there is no common understanding and agreement on how repo versions are going to be used.